DLTtxRecipt/schema.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Minor comment, after '.' please use capital letters. it applies to many descriptions.

Reply:
Besides these comments, it's pending to have the examples and the contributors.yaml and the adopters.yaml. (don't forget to mention the project for dissemination purposes)
thanks a lot.